### DISCUSSION TITLE
Better option names

### DISCUSSION THREAD

**Discussion by seneca** [nuejs/nue#109] [Q&A]
This, does seem like an unfortunate name.
I don't expect to have yes_hotreload.

I think a much simpler name like hot_reload: yes/no should be more than enough.
Of course I may be missing something essential here.

**Comment by tipiirai** [comment]
I see what you mean. Options sound better when positive. I'll change that to plain `hotreload`, which defaults to `true` 

> I don't expect to have yes_hotreload.

You mean that nobody wants to disable hot- reloading in the development mode?

**Comment by seneca** [comment]
Thank you Terio, to be honest, I love the simplicity of Nue and, although not essential, simplifying naming is so in line with the project.

I hope you don't mind me asking you about a couple of other names that you might consider renaming:
no_automount -> automount: yes/no?
page_router -> would router alone not do? will we have other routers in the future?
content_collection -> collection. Do you expect to have other types of collections and if so then this is a great name.
But again, we have collection_name which can be named appropriately if needed.

These are all minor issues but I thought to mention them here. I hope you don't mind that.

**Reply by tipiirai** [reply]
I don't mind at all. I like this conversation, because naming is crucial for simplicity!

- Plain `automount` — yes. Will do. And enabled by default
- Plain `router` — makes sense. There is app router, but it's not configurable via YAML
- Plain `collection` — also makes sense, but not 100% sure. This term was recently introduced by Astro and might be good to be aligned. Thoughts?

**Reply by seneca** [reply]
You are right content_collection is a little bit trickier. This being (under consideration) is the right move.
